### ISSUE TITLE
(3) Grateful Dead Media Collection

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/ucsc-library-digital-collections#380]
### Descriptive Summary

Now that the Grateful Dead media content is now publicly accessible in the DAMS, we are wanting to do some minor enhancements to bring the collection in-line with other collections in the DAMS.

SC&A will identify a new representative image for the media collection, draft a collection description, and select a collection-level rights statement to be used for this set of material.

### Background 

This content was ingested as quick fix to ensure that media could continue to be streamed via GDAO.  Ned did some background work to conceal the collection from public view in the DAMS, but with the creation of the DAMS A-Z Collections pages, the collection is now exposed and accessible.

### Acceptance Criteria

This is what done looks like:

* [x] Representative image updated to Grateful Dead Archive logo
* [x] Collection description added to collection metadata and displays in the About this Collection box on the collection landing page
* [x] Collection-level rights statement added to the collection metadata
* [x] Hide the logo as a work within the collection
* [x] Collection logo should appear as thumbnail in search results

### Related work
Need to complete QA (#472) first; any newly discovered issue will be added to this card.

**Comment by afrebert** [comment]
Issues apparent on page [one](https://digitalcollections.library.ucsc.edu/collections/8k71nh08w?locale=en) during a test run of a generic QA list (ticket #472):
Representative image above not in place
Thumbnail not showing for https://digitalcollections.library.ucsc.edu/concern/works/c821gj76b?locale=en
Thumbnail not showing for https://digitalcollections.library.ucsc.edu/concern/works/th83kz323?locale=en
6 separate files with identical title: Wes Wilson interview. _Interview recorded in 1976 for the documentary radio program "What was that?" broadcast on KSAN-FM in San Francisco [radio broadcast]_

**Comment by rmjaffe** [comment]
Unfortunately as this point, I don't think we can resolve the identical title issue, but it's something to note for when we do the fuller clean up and migration of the collection.

**Comment by rschwab** [comment]
The collection description is in place, but working on this has uncovered some new issues:
- The rights statement has been added to the collection, but they do not appear on the collection landing page. Though I do remember seeing rights statements there (maybe in a mockup?) I can't find any tickets or code that would make them appear on the landing page.
- Adding the logo is not working. I've tried adding the work through the UI and using Bulkrax. The work is created just fine, but the image is not displaying using either upload method.

**Comment by rschwab** [comment]
Will perform a round-trip import on this collection to add rights statement and collection description to each work.

**Comment by rmjaffe** [comment]
@rschwab Just removed "and displays in the About this Collection box on the collection landing page" from the third criterion regarding the addition and display of rights metadata.  I did some digging around on our old project board the the [2020 mockups for the collection description box](https://github.com/UCSCLibrary/dams_project_mgmt/issues/222) include only display of the description. It has since been updated to include the display of harmful language statements.  Chatted with @JessWaggs and given the clean up that needs to be done on the rights info, best not display it at the collection-level for now (if at all).  Rights info will just need to be added and display at the item-level.

**Comment by rmjaffe** [comment]
Blocked by #504

**Comment by rmjaffe** [comment]
@rschwab Seeing a couple things here:

- On the work show pages, the metadata value that is displaying in Copyright Statement (i.e. rightsStatement property, which typically is a rightsStatements.org value) is: http://www.europeana.eu/portal/rights/rr-r.html

I have no idea where this is coming from.

- The free text rights statement (i.e. rightsStatus property) doesn't appear at all in the item metadata:

This work is available from the UC Santa Cruz Library. This digital copy of the work is intended to support research, teaching, and private study. This work is protected by U.S. Copyright Law (Title 17, U.S.C.). Use of this work beyond that allowed by fair use requires written permission of the copyright holder(s). Responsibility for obtaining permission, and for any use or distribution of this work, rests exclusively with the user and not the UC Santa Cruz Library. If the work itself or our research has indicated that one or more individuals or entities is a current copyright holder, that information may be included in the Copyright Information field. Other sources for copyright information include the Creator field or copyright statements on the work. When available, contact information for requesting permission from copyright holder(s) will be linked from the Copyright Information field. If you have additional or conflicting information about ownership of rights in this work, please contact us at [grateful@ucsc.edu](mailto:grateful@ucsc.edu)

**Comment by rmjaffe** [comment]
Unfortunately, still have a couple busted thumbnails too:

![Image](https://user-images.githubusercontent.com/32003407/208977843-462c3886-8d73-4574-a3a9-ceb369321fe7.png)

**Comment by rschwab** [comment]
I can do some round trips to fix these issues.

For the rights statements vs rights status, this is how its described in metadata.yml:
rightsStatement: Information about rights held in and over the resource expressed in general terms (i.e. boilerplate statements).
rightsStatus: An associated license for the object (i.e. Creative Commons license, software licenses, etc.). Defines use/reuse of resources.

As of now, all of the items in this collections have a rightsStatement: Either the "This work is available..." text or the europeana.eu URL. And none of them have a rightsStatus.

How do you want this handled? Should all the works have "This work is available..." for rightsStatement? Should they have anything for rightsStatus?

**Comment by rmjaffe** [comment]
The .yml file is out of sync with the data dictionary (and has been for years).  As far as the rights properties, when working with N8, we had to switch our use and the definitions of rightsStatus and rightStatement per Bulkrax's limitations; this is reflected in the [data dictionary](https://docs.google.com/spreadsheets/d/1q2xrxULg6whSHYxXeirz7KnghEb_xCMPvW6x_wdo9aU/edit#gid=1359124498). Please regard data dictionary as the authority as it's the only version Jess and I know how to update.

For all the works in this collection:
rightsStatement: http://rightsstatements.org/vocab/InC/1.0/
rightsStatus: This work is available from the UC Santa Cruz Library. This digital copy of the work is intended to support research, teaching, and private study. This work is protected by U.S. Copyright Law (Title 17, U.S.C.). Use of this work beyond that allowed by fair use requires written permission of the copyright holder(s). Responsibility for obtaining permission, and for any use or distribution of this work, rests exclusively with the user and not the UC Santa Cruz Library. If the work itself or our research has indicated that one or more individuals or entities is a current copyright holder, that information may be included in the Copyright Information field. Other sources for copyright information include the Creator field or copyright statements on the work. When available, contact information for requesting permission from copyright holder(s) will be linked from the Copyright Information field. If you have additional or conflicting information about ownership of rights in this work, please contact us at [grateful@ucsc.edu](mailto:grateful@ucsc.edu)

Sorry for the confusion!

**Comment by rschwab** [comment]
No worries! Ok the importer is finally working as expected, and the works in this collection are in process of being corrected.

The missing thumbnails are for mp4 assets, so I'm guessing this is going to be a larger issue related to video content. I recommend we open a new ticket for dealing with that.

**Comment by rmjaffe** [comment]
Thanks for creating these additional tickets!  Let me know when the GDAO stuff is ready to be looked at again.

**Comment by rschwab** [comment]
It's ready now, the import has completed.

**Comment by rmjaffe** [comment]
Cool!  A couple observations --  

- Metadata values (rightsStatement, rightsStatus, and subjectName) are showing up on the works (or at least the handful that I looked at) ✓
- When searching, the collection comes up as a matching collection ✓

Question:  Is it possible to hide the logo as a work within the collection, or would that prevent it from serving as the thumbnail image in the about the collection box?
<img width="1249" alt="Screen Shot 2022-12-22 at 12 04 50 PM" src="https://user-images.githubusercontent.com/32003407/209217533-11960ad6-6d47-4392-8a64-1267b6e804f9.png">

**Comment by rmjaffe** [comment]
Just noticed too on the [search result page](https://digitalcollections.library.ucsc.edu/catalog?utf8=%E2%9C%93&search_field=all_fields&q=grateful+dead) that the collection itself might be displays next to the logo, but without the logo appearing as the thumbnail for the collection.

![Image](https://user-images.githubusercontent.com/32003407/209221016-4b7572e2-aed0-4c27-aa0f-5cc3aedf70bf.png)

**Comment by rschwab** [comment]
I updated the acceptance criteria and put this back in "to do". It's currently blocked by #541.

**Comment by rschwab** [comment]
@rmjaffe ready for QA in prod.

**Comment by rmjaffe** [comment]
Logo's showing up in the About the Collection box and when the collection appears as a search result.  Looks good.  Still something afoot with a couple of the thumbnails on the collection landing page: Grateful Dead Met Center show report and Touch of Grey.  Are these video objects?

**Comment by rschwab** [comment]
Yes those are both videos. I've opened ticket #539 to handle this as it seems video as a whole needs more involved work.
